SDL/FxCopSetup.ps1

```diff
@@ -0,0 +1,63 @@
+﻿[CmdletBinding()]
+Param(
+    # C:\Repos\bin\Debug\Src
+    [string]$buildDirectory = (Join-Path -Path (Split-Path -parent (Split-Path -parent (Split-Path -parent $PSCommandPath))) -ChildPath "bin\Debug") ,
+    
+    # C:\Repos\fxCop
+    [string]$fxCopDirectory = (Join-Path -Path (Split-Path -parent (Split-Path -parent (Split-Path -parent $PSCommandPath))) -ChildPath "fxCop")
+)
+
+# these are dlls that end up in the bin, but do not belong to us and don't need to be scanned.
+$excludedFiles = @();
+
+Write-Host "`nPARAMETERS:";
+Write-Host "`tbuildDirectory:" $buildDirectory;
+Write-Host "`tfxCopDirectory:" $fxCopDirectory;
+
+$frameworks = @("net45", "net46", "netstandard1.3");
```

Comment:
would it make sense to move the list of target frameworks to a common props file and have it refered from there? then the main .csproj can also refer from there..
Its easy to forget to update these ps scripts when we update list of frameworks.

Reply:
I didn't see your comment until after I merged.
I didn't consider that because I was testing all these scripts locally where i could inspect each directory. 
But yes, i think that would be a good idea.